internal/cmd/commands/scopescmd/list_key_version_destruction_jobs.go

```diff
@@ -0,0 +1,156 @@
+package scopescmd
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/api/scopes"
+	"github.com/hashicorp/boundary/internal/cmd/base"
+	"github.com/mitchellh/cli"
+	"github.com/mitchellh/go-wordwrap"
+	"github.com/posener/complete"
+)
+
+var (
+	_ cli.Command             = (*ListKeyVersionDestructionJobsCommand)(nil)
+	_ cli.CommandAutocomplete = (*ListKeyVersionDestructionJobsCommand)(nil)
+)
+
+type ListKeyVersionDestructionJobsCommand struct {
+	*base.Command
+}
+
+func (c *ListKeyVersionDestructionJobsCommand) Synopsis() string {
+	return wordwrap.WrapString("List all pending key version destruction jobs within a scope", base.TermWidth)
+}
+
+func (c *ListKeyVersionDestructionJobsCommand) Help() string {
+	return base.WrapForHelpText([]string{
+		"Usage: boundary scopes list-key-version-destruction-jobs [args]",
+		"",
+		"  List all pending key version destruction jobs within a scope. A key version",
+		"  destruction job asynchronously re-encrypts any existing data with the latest",
+		"  key version before destroying the key version. Example:",
```

Comment:
I don't know if this is the right place to explain this sort of thing, but I figured it couldn't hurt, right?